python/pyarrow/_dataset.pyx

```diff
@@ -2406,6 +2428,10 @@ cdef class Scanner(_Weakrefable):
             The maximum row count for scanned record batches. If scanned
             record batches are overflowing memory then this method can be
             called to reduce their size.
+        batch_readahead : int, default 16
+            The number of batches to read ahead in a file. This might not work
+            for all file formats like CSV. Increasing this number will increase
+            RAM usage but also improve IO utilization.
```

Comment:
```suggestion
            The number of batches to read ahead in a file. This might not work
            for all file formats. Increasing this number will increase
            RAM usage but could also improve IO utilization.
```